dtfs-central-api/src/services/repositories/utilisation-reports-repo/update-report-status.ts

```diff
@@ -87,7 +87,8 @@ export const updateManyUtilisationReportStatuses = async (
         }
         return createReportAndSetAsCompleted(utilisationReportsCollection, filter, uploadedByUserDetails);
       case UTILISATION_REPORT_RECONCILIATION_STATUS.REPORT_NOT_RECEIVED:
-        return setToNotReceivedOrDeleteReport(utilisationReportsCollection, filter);
+      case UTILISATION_REPORT_RECONCILIATION_STATUS.PENDING_RECONCILIATION:
+        return setToPendingReconciliationOrDeleteReport(utilisationReportsCollection, filter);
```

Comment:
I've updated the logic now, but this is kind of the whole reason why this temporary solution is a pain. If you try to mark a report as "not completed", you could theoretically try to set the state to either `REPORT_NOT_RECEIVED` or `PENDING_RECONCILIATION` and what actually happens in the database depends entirely on whether or not a report is a placeholder. However, I've updated the logic now so that you can only try to set a report to `PENDING_RECONCILIATION` and dtfs central api decides whether or not to delete the report or set its status (as far as the frontend is concerned, it can see either of those states after clicking the button and the page refreshing)